### ISSUE TITLE
Validation attributes on classes decorated with ModelMetadataTypeAttribute are not emitted

### ISSUE THREAD

**Issue by IanKemp** [domaindrivendev/Swashbuckle.AspNetCore#1802]
```csharp
// 1
public partial class MyModel
{
    [Range(1, 20)]
    public int MyProperty { get; set; }

    public bool OtherProperty { get; set; }
}

// 2
[ModelMetadataType(typeof(MyModelMetadata))]
public partial class MyModel
{
    // purposefully nothing
}

// 3
internal class MyModelMetadata
{
    [FromQuery]
    public bool MyProperty { get; set; }

    [FromQuery]
    public bool OtherProperty { get; set; }
}

//
public class MyController : ControllerBase
{
    public Task<IActionResult> MyAction([FromQuery] MyModel model)
    {
    }
}

```

Given the above example, the `RangeAttribute` applied to `MyProperty` will not be present in `swagger.json` as of Swashbuckle 5.5.1. However, ASP.NET Core does pick up this attribute and validate that field as expected.

If I move the `RangeAttribute` to `MyModelMetadata` the correct metadata is emitted, but that is not how I want to structure my code:

* // 1 will be placed in a NuGet package shared by multiple C# projects - I want the data annotations present to allow producers and consumers of these entities to validate them if desired
* // 2 and // 3 will be in my ASP.NET Core application - the attributes applied in this way are MVC-specific, therefore do not belong in a shared project that will be consumed by non-MVC apps

**Comment by IanKemp** [comment]
Hrm, looking further, the test `GenerateSchema_SetsValidationProperties_IfDataAnnotatedViaMetadataType` does pass and is basically the same scenario. I'm thinking it's something to do with how I'm using `FromQueryAttribute`... continuing to investigate.

**Comment by domaindrivendev** [comment]
I think it's a genuine bug which I've literally just pushed a fix for. I think maybe you're looking at an updated version of the test that I just pushed minutes ago 😄

**Comment by domaindrivendev** [comment]
Here's the fix - 110bd60c

**Comment by IanKemp** [comment]
Found a bug - pushed fix with tests in #1804.

Any ETA on release of 5.6.0?
